docs/developer-guide/new-role.rst

```diff
@@ -0,0 +1,50 @@
+.. Copyright (C) 2021 Robin Schneider <ypid@riseup.net>
+.. Copyright (C) 2021 DebOps <https://debops.org/>
+.. SPDX-License-Identifier: GPL-3.0-or-later
+
+Creating a new DebOps role
+==========================
+
+Historically, a new role was created by copying an existing role and doing an
+initial search and replace. Which has the following problems:
+
+- Which role is up to the latest standards and thus suitable for creating the next role?
+- Requires manual work to setup a new role skeleton before work can be started (search and replace).
+- For new contributers, this is even more difficult.
+- The role standards are going to evolve thus the roles need to be updated.
+
+DebOps uses `Copier`__ to help address those issues.
+
+  .. __: https://copier.readthedocs.io/en/stable/
+
+Setup
+-----
+
+- Install copier.
+- Clone https://github.com/ypid/debops-template.git next to the debops monorepo (or symlink it next to it).
+
+Create a new rule
+-----------------
+
+To create a role called ``new_role``, in :file:`ansible/roles`, run:
```

Comment:
I thought about that:

Pro root:

* Does not relay on the task trick to move the playbook outside of the generated role scope.

Pro inside role:

* The role directory acts as the "boundary" for Copier where the `.copier-answers.yml` file can be kept. This file cannot be in the root because it should be possible to manage multiple independent roles in the same monorepo. `copier update` needs a path where it can be called.

If you can demonstrate a better way to integrate this into the Monorepo, please do.

Reply:
yeah... about that.I was kinda tricked by the copier-answers/yml
In our case, copier is more or less a one-shot thing, you change so much stuff when writing a role that you can't really update using copier

Also, if I want to create mutiple roles, the .copier-answer offers me my previous answers, which is usually not what I want